### PR TITLE
Remove default features from reqwest

### PR DESCRIPTION
**Pull request by urschrei**
These include dependencies which require OpenSSL, which we wish to avoid.

### CODE REVIEW

File: src/proj.rs [hunk 1]
```diff
@@ -776,10 +776,9 @@ mod test {
     fn assert_almost_eq(a: f64, b: f64) {
         let f: f64 = a / b;
         assert!(f < 1.00001);
-        assert!(f > 0.99999);
+        assert!(f > 0.99999);    // This test should be disabled by default, as it requires network access
```
Comment (michaelkirk) [inline src/proj.rs:779]: I think moving this comment might have been unintentional?

Comment (urschrei) [reply src/proj.rs:779]: Hmm looks like rustfmt got confused. "We'll fix it in post"

Comment (urschrei) [reply src/proj.rs:779]: Fixed in 828d58e

**Pull request merged**